packages/cfd/src/Helpers/compare-accounts-config.ts

```diff
@@ -9,97 +9,105 @@ import { CFD_PLATFORMS, MARKET_TYPE, CATEGORY, JURISDICTION, REGION, MARKET_TYPE
 // Map the accounts according to the market type
 const getHighlightedIconLabel = (
     trading_platforms: TModifiedTradingPlatformAvailableAccount,
-    selected_region?: string,
-    is_demo?: boolean
+    selected_region?: string
 ): TInstrumentsIcon[] => {
     const market_type = getMarketType(trading_platforms);
     const market_type_shortcode = market_type.concat('_', trading_platforms.shortcode);
-    // Forex for these: MT5 Financial Vanuatu, MT5 Financial Labuan
-    const forex_label =
-        market_type_shortcode === MARKET_TYPE_SHORTCODE.FINANCIAL_LABUAN ||
-        market_type_shortcode === MARKET_TYPE_SHORTCODE.FINANCIAL_VANUATU ||
-        is_demo ||
```

Comment:
we removed the is_demo check from here.
What are we showing incase of demo?

Reply:
according to figma, yes, it is following `Real`